src/scss/layout/_home-projects.scss

```diff
@@ -0,0 +1,82 @@
+.projects {
+    background-color: var(--alice-blue);
+
+    &__header{
+        @include fontStyle(var(--waikawa-gray), 24px);
+        padding: 2.9rem 5rem 2.3rem;
+        margin-left: auto;
+    }
+
+    &__container {
+        display: flex;
+        flex-direction: column;
+        align-items: center;    
+        padding: 0 3.5rem 2.5rem 3.5rem;
+
+        @include breakpoint('md') {
+            display: grid;
+            grid-template-columns: repeat(2, 1fr);
+            justify-items: center;
+        }
+
+        @include breakpoint('lg') {
+            grid-template-columns: repeat(3, 1fr);
+        }
+    }
+
+    &__card {
+        background: var(--white);
+        box-shadow: 0 4px 4px var(--gray-shadow);
+        border-radius: 16px;
+        max-width: 411px;
+        min-height: 470px;
+        margin: 2rem 0;
+
+        @include breakpoint('md') {
+            max-width: 340px;
+        }
+    }
+
+    &__img {
+        width: 100%;
+        height: 230px;
+        border-radius: 16px 16px 0 0;
+        object-fit: cover;
+    }
+
+    &__title {
+        @include fontStyle(var(--waikawa-gray), 20px);
+        line-height: 26px;
+        padding: 2.2rem 1.5rem 1rem 1.5rem;
+    }
+
+    &__descr {
+        @include fontStyle(var(--cool-gray), 16px);
+        font-weight: 400;
+        line-height: 26px;
+        padding: 0 1.563rem 1.8rem 1.563rem;
+        align-self: center;
+    }
+
+    &__link {
+        display: block;
+        @include fontStyle(var(--dodger-blue), 16px);
+        text-align: center;
+        cursor: pointer;
+        padding-bottom: 2.7rem;
+        font-weight: 700;
+    } 
+}
+
+
```

Comment:
Please remove all these extra spaces in the file. Just leave one extra line in the end and that should be it.